src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -1,14 +1,16 @@
 from __future__ import annotations
 
-import re
 from collections.abc import Mapping
 from enum import Enum
 
-from sentry_kafka_schemas.codecs import ValidationError
-
 from sentry.sentry_metrics.configuration import UseCaseKey
 
-MRI_RE_PATTERN = re.compile("^([c|s|d|g|e]):([a-zA-Z0-9_]+)/.*$")
+
+class UseCaseIDVisibility(Enum):
```

Comment:
I would name this something very specific. `UseCaseAPIAccess` or something like that. Otherwise I'm worried this value will be used for multiple different "permissions", and we'll end up with a tangle of difficult to decode logic.

Reply:
Sounds good, is it fine to have the change of the MRI in this PR also? Or do you want me to split it up?

Reply:
The MRI change is fine with me.